packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -483,6 +497,16 @@ onUnmounted(() => {
 </script>
 
 <style scoped>
+.notebook-section:deep(main .notebook-toolbar),
+.notebook-section:deep(main .ai-assistant) {
+	padding-left: var(--gap-medium);
+}
+
+.notebook-section:deep(main) {
+	gap: var(--gap-small);
+	position: relative;
+}
+
```

Comment:
Maybe write a fix me so someone can fix the css rules in the component so its more plug and play

Reply:
will follow up offline.